tactics/cbn.ml

```diff
@@ -225,7 +225,7 @@ struct
     let equal_cst_member x y =
       match x, y with
       | Cst_const (c1,u1), Cst_const (c2, u2) ->
-        Constant.equal c1 c2 && Univ.Instance.equal u1 u2
+        Constant.CanOrd.equal c1 c2 && Univ.Instance.equal u1 u2
       | Cst_proj p1, Cst_proj p2 -> Projection.repr_equal p1 p2
```

Comment:
repr_equal should be deprecated or quotiented too

Reply:
Right.